setup-jest.ts

```diff
@@ -1,9 +1,3 @@
 import { enableFetchMocks } from 'jest-fetch-mock';
-import { Crypto } from '@peculiar/webcrypto';
 
 enableFetchMocks();
-
-// Assign Node's Crypto to global.crypto if it is not already present
```

Comment:
This needs to stay in so the tests can run with Jest. See https://github.com/workos/workos-node/pull/978